src/components/Button/Button.module.scss

```diff
@@ -437,7 +437,6 @@ button.base {
     content: '';
     position: absolute;
     inset: 1px;
-    z-index: -1;
```

Comment:
Seems like it's working well with this removal. I believe I forgot to remove this line after changing the opacity that was the real problem with the hover being above the text at the time.